src/main/java/com/aws/greengrass/mqtt/bridge/model/RouteLookup.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.mqtt.bridge.model;
+
+import com.aws.greengrass.mqtt.bridge.BridgeConfig;
+import com.aws.greengrass.mqtt.bridge.TopicMapping;
+import lombok.NonNull;
+
+import java.util.Map;
+import java.util.Optional;
+import javax.inject.Inject;
+
+public class RouteLookup {
```

Comment:
why do we need this class?

When we're building the routes we should know all their settings already and just pass that into the clients which need to subscribe/unsubscribe right?

Reply:
yeah that makes sense, original reason for the class was to contain the logic for cross-checking routes in topicMapping and mqtt5RouteOptions.  will refactor to store route stuff as a field in message clients